pkg/Makefile

```diff
@@ -19,7 +20,7 @@ test: lint build
 	go test -vet=off -v $(shell go list ./... | grep -Ev "stubs|network" )
 
 testrace: lint build
-	go test -vet=off -v $(shell go list ./... | grep -Ev "stubs|network" )
+	sudo -E ${GOBINARY} test -vet=off -v $(shell go list ./... | grep -Ev "stubs|network" )
```

Comment:
is it needed to run lint with sudo?

Reply:
this sudo is for tests, tests need to run in sudo because of netlink package

Reply:
I think that is only on our local machines, github workflows shouldn't have that issue, if anyone need to run the test they can run it with sudo